### PR TITLE
Add payment path to transaction section

### PR DESCRIPTION
**Pull request by nathunsmitty**
Based off of #18090, so please review that first

### CODE REVIEW

**Comment by chrisnojima** [comment]
@keybase/react-hackers

File: shared/wallets/transaction-details/payment-path.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
+import * as React from 'react'
+import * as I from 'immutable'
+import * as Kb from '../../common-adapters'
+import * as Styles from '../../styles'
+import * as Types from '../../constants/types/wallets'
+
+interface PaymentPathStartProps {
```
Comment (cjb) [inline shared/wallets/transaction-details/payment-path.tsx]: I guess s/interface/type/ makes sense for consistency with the rest of our code

File: shared/wallets/transaction-details/payment-path.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    <Kb.Box style={styles.paymentPathCircleSmall} />
+    <Kb.Text type="BodyBigExtrabold">
+      {props.assetCode}
+      <Kb.Text type="BodySmall">/{props.issuer}</Kb.Text>
+    </Kb.Text>
+  </Kb.Box2>
+)
+
+const PaymentPath = (props: PaymentPathProps) => {
+  return (
```
Comment (cjb) [inline shared/wallets/transaction-details/payment-path.tsx]: can skip `{ return`

File: shared/wallets/transaction-details/payment-path.tsx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    <Kb.Box2 direction="vertical" alignSelf="flex-start" alignItems="flex-start">
+      <PaymentPathStart assetLabel={props.sourceAmount} issuer={props.sourceIssuer} />
+      <Kb.Box style={styles.paymentPathLine} />
+      {props.pathIntermediate.map(asset => {
+        return (
```
Comment (cjb) [inline shared/wallets/transaction-details/payment-path.tsx]: can skip `{ return`

**Pull request merged**